### PR TITLE
Homepage 1st Draft

### PR DESCRIPTION
**Pull request by liu-caro**
-refactor black background to fix scrolling issues
-implement homepage designs

### CODE REVIEW

File: src/styling/HomePage.css [hunk 1]
```diff
@@ -3,27 +3,52 @@
   src: url(Cornerstone.ttf);
 }
 
-.first-row {
-  margin-top: 150px;
+.logotype {
+  font-family: cornerstone;
+  color: #f8ed44;
```
Comment (feliciazhang) [inline src/styling/HomePage.css]: can we use the css vars for this? there's a couple other places in the rest of this file too

File: src/components/HomePage.js [hunk 2]
```diff
@@ -1,17 +1,74 @@
...
+        <LinkContainer to="/projects">
+          <Button className="button" variant="warning" size="lg">
+            Volunteer for Projects
+          </Button>
+        </LinkContainer>
+      </Col>
+    </Row>
+    <Row className="track-titles justify-content-center">Request Project Help</Row>
+    <Row className="justify-content-center">
+      <Image src={business1} thumbnail />
```
Comment (feliciazhang) [inline src/components/HomePage.js:46]: you don't have to change this but a tip for future for less typing is if you just make an array with all the photo vars and do 
```
businessImages.map((img) => (<Image src={image}/>))
```

**Review by feliciazhang — APPROVED** [review]

File: src/styling/HomePage.css (hunk 1 above)
Comment (liu-caro) [reply src/styling/HomePage.css]: fixed!

File: src/components/HomePage.js (hunk 2 above)
Comment (liu-caro) [reply src/components/HomePage.js:46]: good note! might refactor later haha (:

**Pull request merged**